devlog/_plan/260827_igwanu_bug_pr_merge_round/041_wp2b_2729_supersede.md

```
@@ -0,0 +1,161 @@
+claim this repository's standing gates exist to distrust, and I reached for it
+with two facts that did not hold.
+
+## Then the arm itself turned out not to fire
+
+The final audit also found that #2729's `failed_precondition` branch did not
+trigger on the shape it exists to catch. It sat **after** the overload keywords in
+both `classifyCursorError` and `inferHttpStatusFromAdapterMessage`, and a
+plan-gated rejection normally reads `failed_precondition: model unavailable for
+this plan` — which matches `unavailable` first:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)